plugins/woocommerce/src/Utilities/OrderUtil.php

```diff
@@ -196,8 +196,8 @@ public static function get_table_for_order_meta() {
 	public static function get_count_for_type( $order_type ) {
 		global $wpdb;
 
-		$cache_key        = 'order-count-' . $order_type;
-		$count_per_status = wp_cache_get( $cache_key, 'orders' );
+		$cache_key        = 'wc-order-count-' . $order_type;
```

Comment:
This should use the `WC_Cache_Helper::get_cache_prefix( 'orders' )` as the prefix, to tie in with wc order cache.  Passing count as a group `counts` will probably not work on memcache-based clients.
```suggestion
		$cache_key        = WC_Cache_Helper::get_cache_prefix( 'orders' ) . 'wc-order-count-' . $order_type;
```

Reply:
Applied in 94ef10bbb7132d8525b09efe6100c2265f6ebbeb.